test/end-to-end/auth-client.js

```diff
@@ -65,3 +69,18 @@ test('isCompatible', async(t) => {
   t.truthy(teslaComp);
   t.falsy(royceComp);
 });
+
+test('isCompatible - with scopes', async(t) => {
+  const client = new smartcar.AuthClient(getAuthClientParams());
+
+  const teslaVin = '5YJXCDE22HF068739';
+  const audiVin = 'WAUANAF40HN017169';
```

Comment:
Just verifying, this audi is known to be incompatible?

Reply:
yeeep. compatibble with the hardware but not with the scopes provided in the test.